hpack/hpack.py

```diff
@@ -22,6 +22,7 @@
 INDEX_NONE = b'\x00'
 INDEX_NEVER = b'\x10'
 INDEX_INCREMENTAL = b'\x40'
+_PREFIX_BIT_MAX_NUMBERS = [(2 ** i) - 1 for i in range(10)]
```

Comment:
Sure thing, I will reduce it to 9 instead of 10 as well, the 0 index entry is to save us a subtraction on usage.